src/Prettus/Repository/Criteria/RequestCriteria.php

```diff
@@ -46,7 +46,7 @@ public function apply($model, RepositoryInterface $repository)
 
         if ($search && is_array($fieldsSearchable) && count($fieldsSearchable)) {
 
-            $searchFields = is_array($searchFields) || is_null($searchFields) ? $searchFields : explode(';', $searchFields);
+            $searchFields = is_array($searchFields) || is_null($searchFields) ? $searchFields : array_filter(explode(';', $searchFields));
```

Comment:
Querys like this would not get handled in the  `RequestCriteria`. You need to add it to the `?search=...` param, right?